targets/UnrealMarketplacePlugin/makeBp.js

```diff
@@ -307,7 +307,7 @@ function getCustomApiAssignmentLogic(tabbing, apiCall) {
             + tabbing + "TArray<FString> debugFunctions = PlayFabCommon::PlayFabCommonUtils::GetLocalSettingsFileArrayProperty(TEXT(\"DebugFunctions\"));\n"
             + tabbing + "if (!localApiServer.IsEmpty() && (debugFunctions.Contains(TEXT(\"*\")) || debugFunctions.Contains(request.FunctionName)))\n"
             + tabbing + "{\n"
-            + tabbing + "    FString endpoint = TEXT(\"/CloudScript/ExecuteFunction\");\n"
+            + tabbing + "    FString endpoint = TEXT(\"" + apiCall.url + "\");\n"
```

Comment:
Wow, this line is really hard to parse!

Reply:
I don't follow what you are asking me to do here. Can you please explain?

Reply:
I don't think there is anything you can do. Just the combination of the TEXT macro and the content of that macro being generated makes the escaping of the various " characters difficult to read.

Reply:
In that case if every concern on this PR is addressed can we please approve and merge it so we can proceed with handing it to the SDK team for review? I don't suspect them to approve it immediately and so more work is awaiting me waiting on this to get approved.